apps/challenges/views.py

```diff
@@ -2542,6 +2551,44 @@ def validate_challenge_config(request, challenge_host_team_pk):
             )
             error_messages.append(message)
 
+        # To ensure that the schema for submission meta attributes is valid.
+        if data.get("submission_meta_attributes"):
```

Comment:
@KhalidRmb instead of duplicating the code we should create a helper method for the redundant validations. See the https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/challenge_config_utils.py.